### ISSUE TITLE
Functional - General - Footer Icons Do Not Notify TalkBack User They are Link or Button

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/OCIO-DOLSafety-Android#5]
The application does not tell the TalkBack user that the “All Incidents” and the other three icons/items in the footer can be tapped to go to another screen as a link or that they are a button. I think they should be a link since they are going somewhere and not doing a function. But if memory serves me correctly when using  TalkBack, it vibrates/buzzes the phone, and if that is the case, then we are good.

**Comment by greg-gruse** [comment]
This is a duplicate of https://github.com/USDepartmentofLabor/OCIO-DOLSafety-Android/issues/2

**Comment by jfc3-dol** [comment]
Closing issue as a duplicate.
